okio-nodefilesystem/build.gradle

```diff
@@ -0,0 +1,58 @@
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+
+kotlin {
+  if (kmpJsEnabled) {
+    js {
+      configure([compilations.main, compilations.test]) {
+        tasks.getByName(compileKotlinTaskName).kotlinOptions {
+          moduleKind = "umd"
+          sourceMap = true
+          metaInfo = true
+        }
+      }
+      nodejs {
+        testTask {
+          useMocha {
+            timeout = "30s"
+          }
+        }
+      }
+    }
+  }
+  sourceSets {
+    all {
+      languageSettings {
+        useExperimentalAnnotation('kotlin.RequiresOptIn')
+      }
+    }
+    jsMain {
+      dependencies {
+        implementation project(":okio")
+        api deps.kotlin.stdLib.common
+        // Uncomment this to generate fs.fs.module_node.kt. Use it when updating fs.kt.
+        // implementation(npm("@types/node", "14.14.16", true))
+        api deps.kotlin.stdLib.js
+      }
+    }
+    jsTest {
+      dependencies {
+        implementation deps.kotlin.test.common
+        implementation deps.kotlin.test.annotations
+        implementation deps.kotlin.time
+
+        implementation project(":okio-fakefilesystem")
+        implementation deps.kotlin.test.js
+      }
+    }
+  }
+}
+
+
+// modify these lines for MANIFEST.MF properties or for specific bnd instructions
+project.ext.bndManifest = '''
+    Export-Package: okio.nodefilesystem
+    Automatic-Module-Name: okio.nodefilesystem
+    Bundle-SymbolicName: com.squareup.okio.nodefilesystem
+    '''
```

Comment:
Can be deleted. No OSGi nonsense in the leftpad ecosystem.

Reply:
NICE yes!